katas/content/multi_qubit_gates/index.md

```diff
@@ -121,6 +121,82 @@ $$\alpha|00\rangle + \beta|11\rangle$$
 
 The $CNOT$ gate is self-adjoint: applying it for the second time reverses its effect.
 
+
+@[exercise]({
+    "id": "multi_qubit_gates__two_qubit_gate_1",
+    "title": "Two Qubit Gate 1",
+    "path": "./two_qubit_gate_1/",
+    "qsDependencies": [
+        "../KatasLibrary.qs"
+    ]
+})
+@[section]({
+    "id": "multi_qubit_gates__cz_gate",
+    "title": "CZ Gate"
+})
+
+## CZ Gate (Controlled-Z Gate)
+
+The $CZ$ ("controlled-Z") gate is a two-qubit gate, with one qubit referred to as the **control** qubit, and the other as the **target** qubit. Typically, the first qubit is the control, and the second qubit is the target.
+
+The $CZ$ gate acts as a conditional gate: if the control qubit is in state $|1\rangle$, it applies the $Z$ gate to the target qubit, otherwise it does nothing.
+
+> If the system is in a superposition of several basis states, the effects of the gate will be a linear combination of the effects of it acting separately on each of the basis states.
+> This is a common feature for all quantum gates you'll encounter that are specified in terms of basis states: since all unitary gates are linear, it is sufficient to define their effect on the basis states, and use linearity to figure out their effect on any state.
+
```

Comment:
I don't think we need to repeat this note, since we mention for CNOT that this is a common feature
```suggestion
```